forge/routes/auth/oauth.js

```diff
@@ -169,12 +169,22 @@ module.exports = async function (app) {
                     code: requestObject.code,
                     state: requestObject.state
                 })
+                console.log('/account/complete redirect', responseUrl.toString())
                 reply.redirect(responseUrl.toString())
                 return
             }
         }
         return badRequest(reply, 'access_denied', 'Access Denied')
     })
+    app.get('/account/reject/:code', async function (request, reply) {
```

Comment:
This slipped in and isn't strictly part of the PR - but will be part of the next PR in this area :)